packages/@functionless/asl-graph/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@functionless/asl-graph",
+  "version": "0.27.0",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib",
+    "docs"
+  ],
+  "scripts": {
+    "build:docs": "fls-typedoc",
+    "clean:build": "rm -rf lib tsconfig.tsbuildinfo",
+    "typecheck": "tsc --noEmit",
+    "dev": "tsc --watch"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-appsync-alpha": "2.44.0-alpha.0",
+    "@functionless/ast": "^0.27.0",
+    "@functionless/aws-stepfunctions": "^0.27.0",
+    "@functionless/util": "^0.27.0",
+    "aws-cdk-lib": "2.44.0",
```

Comment:
I am fairly sure this will cause an issue for consumers who want to use other versions, but we can try it.

They will get two versions of CDK unless they force using `resolutions`